src/Utility/Text.php

```diff
@@ -852,4 +859,75 @@ public static function parseFileSize($size, $default = false)
         }
         throw new InvalidArgumentException('No unit type.');
     }
+
+    /**
+     * Get/set default transliterator identifer string.
+     *
+     * @param string|null $transliteratorId Transliterator identifer.
+     * @return string|void
+     */
+    public static function defaultTransliteratorId($transliteratorId = null)
+    {
+        if ($transliteratorId === null) {
+            return static::$_defaultTransliteratorId;
+        }
+
+        static::$_defaultTransliteratorId = $transliteratorId;
```

Comment:
returning the string sounded fine to me (to actually read it), just not null as default.
But if we use separate getters/setters here this would be resolved in the process.